docs/configuration/settings.rst

```diff
@@ -5,113 +5,469 @@ Mautic Configuration Settings
+
+   Ensure that you configure your sender domain, DKIM, bounce, and click tracking domains. For more information, see Email domains document.
+
+* **Reply to address** -  To have Contacts reply to a different address than the address defined in Email address to send mail from, add the desired address to receive replies. This is the default address where messages are sent whenever Contacts reply to messages sent from Mautic. If this field is blank, the address specified in **Email address to send mail** from is used. This is useful if your configured sender domain, which you use in the from address, contains a subdomain that doesn't have MX records or is otherwise an address that can't receive Emails.
+
+* **Custom return path (bounce) address** - Set a custom return path/bounce Email for Emails sent from the system. Note that some mail transports, such as GMail, won'the support this.
+
+* **Mailer is owner** - If Contacts in Mautic have ``Owners``, select Yes to display the owner as sending all Emails to the Contacts they’re listed as the owner for. 
+
+.. note:: 
+
+    Mailer is owner overrides any other name or Email to send mail from, including the default and individual Emails. Every Contact owner’s domain must have SPF and DKIM records. You can see this configuration for individual Emails, rather than globally.
+    For more information see :doc:`Mailer is owner</channels/emails>`
+
+* **Service to send mail through** - For most Mautic customers, this should remain as ``Mautic``. Your Emails route through the Mautic's Sparkpost account. If you are sending through your own Email service, select the service and enter your credentials. 
+  
 Default frequency rule
 ======================
 
+* **Do not contact more than <number> each <period>** - This limits the number of marketing messages a Contact receives in a certain period of time (day, week, month). Transactional messages don’t count towards this limit. You can adjust this at the individual Contact level, either manually or by preference center setting. 
+
+*More information is available in :doc:`Default Frequency Rule document</contacts/frequency_rules>`.*
+
 Monitored inbox settings
 ========================
 
+.. image:: images/monitored-settings.png
+  :width: 600
+  :alt: Screenshot showing Monitored Settings Configuration in Mautic
+
+* **Default Mailbox** - If Email addresses your messages are going to bounce, this inbox is where you receive those bounce notifications.
+
+* **Bounces** - A folder to monitor for new bounce messages or Emails.
+
+* **Unsubscribe Requests** - A folder to monitor for new unsubscribe requests. 
+
+* **Contact Replies** - Similar to the monitored inbox for bounces, this is the inbox Mautic checks for Contact replies. Using "Replies to Email" decisions in any Campaign requires configuration. With ``Use custom connection settings?`` set to ``no``, Mautic checks the default mailbox. If set to ``yes``, you may track a different mailbox for replies.
```

Comment:
Can we link to the docs about the Replies to Email configuration? Create a blank section if it is not yet written.

Reply:
Okay